packages/ui-popover/src/Popover/README.md

```diff
@@ -105,21 +152,136 @@ render: false
 example: true
 ---
 class Example extends React.Component {
-render () {
-  return (
-      <div style={{ paddingBottom: 25, display: 'flex', justifyContent: 'center' }}>
-        <Popover
-          renderTrigger={<div style={{display: 'inline-block', height: '3px', width: '3px', background: 'blue'}}/>}
-          isShowingContent={true}
-          placement="end top"
-          shouldAlignArrow
-          mountNode={() => document.getElementById('main')}
-        >
-          <Heading>Small<br/>Target</Heading>
-        </Popover>
-      </div>
-  )
-}
+    constructor(props) {
+        super(props)
+
+        this.state = {
```

Comment:
You only should store those variables in state, which you'd like to change inside the component. For example placementsValues is a constant list, it will never change. So is ShadowValues. They should be a static variable on the class.

Reply:
I will fix it and create pull request again